rundeckapp/grails-app/init/rundeckapp/Application.groovy

```diff
@@ -65,6 +61,17 @@ class Application extends GrailsAutoConfiguration implements EnvironmentAware {
         loadGroovyRundeckConfigIfExists(environment)
     }
 
+    @Override
+    void doWithApplicationContext() {
+        if(rundeckConfig.isRollback()) {
+            RundeckDbMigration rundeckDbMigration = new RundeckDbMigration(applicationContext, grailsApplication)
```

Comment:
Does the RundeckDbMigration class need a grailsApplication object passed to it? I didn't see it being used.